website/docs/r/lambda_permission.html.markdown

```diff
@@ -151,3 +151,11 @@ resource "aws_lambda_permission" "lambda_permission" {
  * `statement_id_prefix` - (Optional) A statement identifier prefix. Terraform will generate a unique suffix. Conflicts with `statement_id`.
 
 [1]: https://developer.amazon.com/docs/custom-skills/host-a-custom-skill-as-an-aws-lambda-function.html#use-aws-cli
+
+## Import
+
+Lambda Function Aliases can be imported using the function_name/statement_id, e.g.
```

Comment:
I think this should read something like: 
```suggestion
Lambda permission statements can be imported using the function_name/statement_id, e.g.
```
Could you double-check that please?

Reply:
Addressed f925fb2